packages/core/cms/faststore/sections.json

```diff
@@ -304,6 +304,24 @@
                       }
                     }
                   }
+                },
```

Comment:
Do you think we should move this to our globalSettings? (Not sure if we will be re-using this info in more places 🤔)